feedhenry/fh-fhc
# task(Blackberry deprecations): Remove the texts about it

# Jira link(s)
https://issues.jboss.org/browse/RHMAP-21172

# What
Remove the Blackberry refs. 

# Why
It is deprecated and should not so long be in the NGUI or FHC

# How
Updating the bundles and outputs. 

# Verification Steps

Run the fhc build and check if the Blackberry info was removed from there. 
```

cmacedo@camilas-MBP ~/work/fh-fhc (RHMAP-21172) $ ./bin/fhc.js build
Usage:
 fhc build --project=<project> --app=<app> --cloud_app=<cloud_app> [--tag=<tag>]
 --environment=<environment> --destination=<destination> [--version=<version>]
 [--config=<config>] [--privateKeyPass=<privateKeyPass>] [--certpass=<certpass>]
 [--download=<download>] [--bundleId=<bundleId>]
 [--cordova_version=<cordova_version>] [--git-tag=<git-tag>]
 [--git-branch=<git-branch>] [--git-commit=<git-commit>] [--json=<json>]

Options:
  --help                   Show help                                   [boolean]
  --version, -v            Specific to the destination (e.g. Android version
                           4.0).                                       [boolean]
  --project, -p            Unique 24 character GUID of the project.   [required]
  --app, -a                Unique 24 character GUID of the client app which will
                           be built.                                  [required]
  --cloud_app, --ca        Unique 24 character GUID of the cloud app which the
                           client app will connect to.                [required]
  --tag, -t                The name of a connection tag for the cloud app, must
                           be in Semantic Version format, e.g. 0.0.1. See:
                           http://semver.org.
  --environment, -e        The id of a target environment, e.g. dev.  [required]
  --destination, -d        One of: android, iphone, ipad, ios(for universal
                           binary)                                    [required]
  --config, -c             Either 'debug' (default), 'distribution', or
                           'release'.
  --privateKeyPass, --pk   Private key password of the bundleId, only needed for
                           'release' builds.
  --certpass, --cp         Certificate password.
  --download, --dw         Boolean value to inform to download or not the built
                           when it finish.
  --bundleId, -b           The unique bundle identifier of the credential
                           bundle. You can get it using 'fhc credentials list'.
                           Required for all iOS builds.
  --cordova_version, --cv  For specifying which version of Cordova to use.
                           Currently supported: either 2.2 or 3.3. Only valid
                           for Android for now.
  --git-tag, --gt          Long value of URL that you want shorter for the
                           henr.ie URL.
  --git-branch, --gb       The name of the git branch to build, defaults to
                           'master'.
  --git-commit, --gc       The full hash of the commit to build, if not the head
                           of the branch. Must be used with <branch name>.
  --json, -j               Output into json format


```

## Checklist:
